pages/events/index.vue

```diff
@@ -327,13 +327,6 @@ export default class EventsPage extends QiskitPage {
       margin-top: $layout-02;
       margin-bottom: $layout-03;
     }
-
-    &__cta {
```

Comment:
You forgot to remove the [line where this class was used](https://github.com/Qiskit/qiskit.org/pull/1145/files#diff-e89e9da7098c8b828fffda21f9b905c5668d8726f92332ddd3612c55d7bc0d9eR77)